src/state/lists/updater.ts

```diff
@@ -28,6 +29,12 @@ export default function Updater(): null {
     )
   }, [fetchList, isWindowVisible, lists])
 
+  useEffect(() => {
+    if (chainId && [SupportedChainId.OPTIMISM, SupportedChainId.OPTIMISTIC_KOVAN].includes(chainId)) {
+      dispatch(enableList(OPTIMISM_LIST))
```

Comment:
this isn't exactly what was discussed, which was to add to the default list of lists and enhance the UI to indicate which lists have no tokens on the current chain. did those requirements change?

Reply:
if you remove this part you can address the token lists ux changes in a separate pr so this very broad pr isn't outstanding for too long (and just add optimism list to default list of lists)